fileHub/js-app/app/components/component.js

```diff
@@ -48,6 +49,15 @@ export class Component {
     return this.rootElement.querySelectorAll(`[data-fh="${searchDataAttribute}"]`);
   }
```

Comment:
This element returns not an HTMLElement but a collection of it (not array). So maybe it is better to transform it to array here and return `HTMLElement[]`.